t/tests.py

```diff
@@ -10,50 +10,116 @@
 
 api = sepsesam.api.Api(**cred, verify=False)
 
-
 class TestSepSesam(unittest.TestCase):
     """ sepsesam.api unittests """
 
-    """ server info """
 
-    def testGetServerInfo(self):
-        self.assertEqual(api.get_server_info()["restPort"], 11401)
+    """Managing Backup Tasks"""
+
+    
+
```

Comment:
Please consider using a formatter like [Ruff](https://marketplace.visualstudio.com/items?itemName=charliermarsh.ruff) for auto-formatting. These line breaks are huuge :)